api/SubmitJob/src/main.py

```diff
@@ -0,0 +1,27 @@
+import json
+from os import getenv
+
+import boto3
+
+client: boto3 = boto3.client('batch')
+
+
+def lambda_handler(event, context):
+    parameters = json.loads(event['body'])
+    job = client.submit_job(jobName=parameters['granule'],
+                            jobQueue=getenv('JobQueue'),
+                            jobDefinition=getenv('JobDefinition'),
+                            parameters={
```

Comment:
this can just be `parameters=parameters`, yes?  lets the function itself stay ignorant of the parameters for a particular processing type?